### PR TITLE
feat: enable theming notifciation (ACQ-1683)

### PR DESCRIPTION
**Pull request by 0ubbe**
## Summary 💭

- Make **notification styles** handle light/dark theme
- Enable customising notification through `theme.notification`
- Move **debugger** to the bottom ( so it doesn't collide with notification )
- Improve **notification styles** by giving at bg so it doesn't overlap with card bg

### Regular 

Screenshot 2021-08-06 at 17 36 57

Screenshot 2021-08-06 at 17 37 11

### Debugger enabled

Screenshot 2021-08-06 at 17 36 37

Screenshot 2021-08-06 at 17 36 20

### CODE REVIEW

**Review by IagoLast — APPROVED** [review]

File: packages/pci-sdk-iframe/src/pages/app/themes/dark.ts
Comment (alejandraarri) [inline packages/pci-sdk-iframe/src/pages/app/themes/dark.ts:103]: I would change the CSS a bit if you don't mind, so we keep using relative units in those rules that need it and better follow the styleguide guidelines for branding consistency.

```suggestion
	position: 'absolute',
	zIndex: 3,
	top: 4.706vw;
	left: '50%',
	width: '70%',
	padding: '2vw 4vw',
	transform: 'translateX(-50%)',	
	borderRadius: '8px',
	backgroundColor: '#333333',
	color: '#CFD8DC',
	fontFamily: 'sans-serif',
	font-size: 4.118vw;
  line-height: 1.5;
  textAlign: 'center',
```

File: packages/pci-sdk-iframe/src/pages/app/themes/light.ts
Comment (alejandraarri) [inline packages/pci-sdk-iframe/src/pages/app/themes/light.ts:100]: The styles for the light theme would be the following:
```suggestion
	position: 'absolute',
	zIndex: 3,
	top: 4.706vw;
	left: '50%',
	width: '70%',
	padding: '2vw 4vw',
	transform: 'translateX(-50%)',	
	borderRadius: '8px',
	backgroundColor: '#CBCBCB',
	color: '#CFD8DC',
	fontFamily: 'sans-serif',
	font-size: 4.118vw;
  line-height: 1.5;
  textAlign: 'center',
```

**Review by alejandraarri — COMMENTED** [review]
Overall looks good, left some CSS change suggestions. ✨ 

I have another PR in review that is waiting for your approval that also fixes this issue. https://github.com/AptoPayments/apto-pci-sdk-web/pull/51 Which one are we going to include in the next release?

**Pull request merged**
